app/src/main/res/values-iw/strings.xml

```diff
@@ -47,8 +47,7 @@
   <string name="error_rendering_markdown">עיבוד הוזלה נכשל</string>
   <string name="error_users_search">חיפוש משתמשים נכשל</string>
   <string name="loading_gist">טוען תמצית…</string>
-  <string name="random_gist">טוען תמצית אקראית…</string>
-  <string name="loading_more_issues">טוען עוד סוגיות…</string>
+    <string name="loading_more_issues">טוען עוד סוגיות…</string>
```

Comment:
Really minor issue, that should not be indented like that

Reply:
Nice catch! Fixed it